app/views/ctc/questions/dependents/does_not_qualify_ctc/edit.html.erb

```diff
@@ -9,7 +9,7 @@
     <p><%= puerto_rico_filing ? t("views.ctc.questions.dependents.does_not_qualify_ctc.puerto_rico.help_text", name:  @dependent.first_name.upcase_first) : t("views.ctc.questions.dependents.does_not_qualify_ctc.help_text", name:  @dependent.first_name.upcase_first) %></p>
     <%= render('components/molecules/reveal', title: t("views.ctc.questions.dependents.does_not_qualify_ctc.reveal")) do %>
       <ul class="list--bulleted">
-        <% conditions = puerto_rico_filing ? t("views.ctc.questions.dependents.does_not_qualify_ctc.puerto_rico.conditions", current_tax_year: current_tax_year, filing_year: current_tax_year + 1) : t("views.ctc.questions.dependents.does_not_qualify_ctc.conditions", current_tax_year: current_tax_year, filing_year: current_tax_year + 1) %>
+        <% conditions = puerto_rico_filing ? t("views.ctc.questions.dependents.does_not_qualify_ctc.puerto_rico.conditions", current_tax_year: MultiTenantService.new(:ctc).current_tax_year, filing_year: ctc_current_tax_year + 1) : t("views.ctc.questions.dependents.does_not_qualify_ctc.conditions", current_tax_year: MultiTenantService.new(:ctc).current_tax_year, filing_year: ctc_current_tax_year + 1) %>
```

Comment:
i think we already discussed doing this in a followup but since `MultiTenantService.new(:ctc).current_tax_year` and `ctc_current_tax_year` are (i think) the same thing it would be cool to be consistent in how we reference them

Reply:
Agreed. I'd prefer to defer to a follow-up commit. The big question I think is, Should we expose `ctc_service` as a helper, or expose `ctc_current_tax_year`? The former is better if we need other properties like `.host`. The latter is better if we only need the year.